compiler/optimizer/OMRTransformUtil.cpp

```diff
@@ -390,7 +390,8 @@ OMR::TransformUtil::removeTree(TR::Compilation *comp, TR::TreeTop * tt)
 void
 OMR::TransformUtil::transformCallNodeToPassThrough(TR::Optimization* opt, TR::Node* node, TR::TreeTop * anchorTree, TR::Node* child)
    {
-   opt->anchorAllChildren(node, anchorTree);
+   if (anchorTree)
```

Comment:
Why leave the door open for a functional bug by not forcing the anchoring of children ?

Reply:
@vijaysun-omr If the call can be transformed to a passthrough, its children are not needed. Maybe the anchoring is never needed for this transformation.

Reply:
I'm not convinced yet. The nature of what is being transformed does'nt affect whether we can safely delete the children (having reference count > 1) or not, unless you are sure that the children are all ONLY accessed from the treetop being transformed OR the treetop in question is guaranteed to be later than the first evaluation points for those children.

Reply:
No, those are not guaranteed. In the case of unnecessary anchoring, other opts can easily get rid of dead trees if the anchoring is not needed. I'll close this PR as per discussion.